tests/v2/validation/prime/prime_test.go

```diff
@@ -0,0 +1,68 @@
+package prime
+
+import (
+	"encoding/json"
+	"strconv"
+	"testing"
+
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	"github.com/rancher/rancher/tests/framework/extensions/rancherversion"
+	"github.com/rancher/rancher/tests/framework/pkg/config"
+	"github.com/rancher/rancher/tests/framework/pkg/session"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+)
+
+const rancherBrand = "suse"
+
+type PrimeVersionTestSuite struct {
+	suite.Suite
+	session        *session.Session
+	client         *rancher.Client
+	isPrime        bool
+	rancherVersion string
+}
+
+func (t *PrimeVersionTestSuite) TearDownSuite() {
+	t.session.Cleanup()
+}
+
+func (t *PrimeVersionTestSuite) SetupSuite() {
+	testSession := session.NewSession(t.T())
+	t.session = testSession
+	primeConfig := new(Config)
+
+	config.LoadConfig(ConfigurationFileKey, primeConfig)
+	t.isPrime = primeConfig.IsPrime
+	t.rancherVersion = primeConfig.RancherVersion
+
+	client, err := rancher.NewClient("", t.session)
+	assert.NoError(t.T(), err)
+	t.client = client
+
+}
+
+func (t *PrimeVersionTestSuite) TestPrimeVersion() {
+	byte_object, err := rancherversion.RequestRancherversion(t.client.RancherConfig.Host)
+	assert.Nil(t.T(), err)
+
+	var jsonObject map[string]interface{}
```

Comment:
The unmarshaling from 49-51 should be in the extension you wrote. I don't think logging the json brings any value, maybe the version so when it fails or passes we know what version it is.